src/abb_system_position_only.cpp

```diff
@@ -29,6 +29,9 @@ using namespace std::chrono_literals;
 
 namespace ros2_control_abb_driver
 {
+
+static constexpr auto DEG_TO_RAD = 180 / M_PI;
```

Comment:
Please put this in an anonymous namespace so it doesn't pollute the `ros2_control_abb_driver` namespace. An anonymous namespace keeps the constant local to this file. Also, I've found out from Tyler recently, the `static` does not do anything here.

```
namespace
{
  constexpr auto DEG_TO_RAD = 180 / M_PI;
}
```

Reply:
I'm pretty sure static / anonymous namespace do the same thing (see [here](https://stackoverflow.com/questions/154469/unnamed-anonymous-namespaces-vs-static-functions)), but if we in general prefer anonymous namespaces for stylistic reasons, I'm fine with changing it.

Reply:
OK, good point. Leave it as-is, then